pub/refget-openapi.yaml

```diff
@@ -159,7 +159,8 @@ components:
             ga4gh:
               type: string
               description: >-
-                A ga4gh identifier used to identify the sequence. This is a base64url-encoded encoded
+                A ga4gh identifier used to identify the sequence. This is a SHA-512 
+                [base64 url encoded](https://datatracker.ietf.org/doc/html/rfc4648#section-5) 
                 string taking the 1st 24bits from the SHA-512 digest (sha512t24u). This is the preferred method
```

Comment:
The first 24 **bytes**.

It's also not clear here  that `sha512t24u` refers to the entire SHA-512 (`sha512`) truncated to 24 bytes (`t24`) base64url-encoded (`u`) pipeline.

Reply:
Eeep. Yes bytes. Okay

Reply:
Slightly astonished neither I nor anyone else had noticed that before! :smile: